### PR TITLE
DS_Week3_복습과제_황선경

### PR DESCRIPTION
**Pull request by mongshil**

### CODE REVIEW

**Comment by kkyung0131** [comment]
1.3
.head() 대신 .columns() 로 더 간편하게 확인하신 것 같아요!
감사합니다!

1.6 
더 간단하게 인코딩하신 것 같아요!
레이블 인코딩을 할 수 있는 다른 코드도 보시면 좋을 것 같아 적어드립니다!
1)(파머완 코드)
items = ['white','red']
encoder = LabelEncoder()
encoder.fit(items)
labels = encoder.transform(items)
train_df['type'] = encoder.fit_transform(train_df['type'].values)
print(train_df['type'].value_counts())

2)(차수빈벗 코드)
encoder = LabelEncoder() # 라벨 인코더 객체 생성
encoder.fit(train_df['type'])
train_df['type'] = encoder.transform(train_df['type']) # label-encoding을 시킨 결과를 다시 df의 컬럼에 저장
train_df['type'].value_counts()
# red, white -> 0,1로 encoding

3)(조혜빈벗 코드)
items=['type']
le=LabelEncoder()
for i in items:
traindrop[i]=le.fit_transform(traindrop[i])
traindrop['type'].value_counts()  

2.5
저랑 정확도가 다르게 나와서 살펴봤더니, random_state를 2.4번과는 다르게 지정 안해주셔서 그런 것 같아요! 지금 있는 코드에서 random_state=12만 추가해주시면 될 것 같습니다!!
rf_clf1 = RandomForestClassifier(n_estimators=50, min_samples_leaf=8,max_depth=14,
min_samples_split=8, random_state=12)
2.6
답이랑 선경님 그래프랑 차이점은 순위순으로 배치되어 있느냐? 아니냐? 인 것 같아요. 문제에는 피처 주요도를 막대 그래프로 시각화 하라고 했지, 순위순으로 하라고 안했기 때문에 답이랑 다르다고 해서 신경쓰지 않아도 될 것 같습니다!
3.2
문제에 언급된 부분이 없음에도 cv를 설정해주신 점 좋아요!

3.3
best_estimator_를 이용하는 방법 말고 다른 방법도 있어서 적어드립니다.
lin_svm = SVC(kernel='linear',C=1)
lin_svm.fit(X_train,y_train)
pred = lin_svm.predict(X_test)
accuracy = accuracy_score(y_test, pred)
print('accuracy : {0:.4f}'.format(accuracy)) 

3.4
중간에 오타가 있는데 혹시 오류가 나지는 않으셨나요? 대소문자 오타 조심하세요!!
